app_flutter/lib/widgets/task_icon.dart

```diff
@@ -44,22 +44,19 @@ class TaskIcon extends StatelessWidget {
           color: blendFilter,
         );
       case StageName.luci:
-        return Image.asset(
-          'assets/chromium.png',
-        );
-      case StageName.devicelab:
-        return Image.asset(
-          'assets/android.png',
-        );
-      case StageName.devicelabWin:
-        return Image.asset(
-          'assets/windows.png',
-        );
-      case StageName.devicelabIOs:
-        return Image.asset(
-          'assets/apple.png',
-          color: blendFilter,
-        );
+        if (qualifiedTask.builder.toLowerCase().startsWith('linux')) {
+          return Image.asset(
+            'assets/linux.png',
+          );
+        } else if (qualifiedTask.builder.toLowerCase().startsWith('mac')) {
+          return Image.asset(
+            'assets/apple.png',
+          );
+        } else if (qualifiedTask.builder.toLowerCase().startsWith('win')) {
+          return Image.asset(
+            'assets/windows.png',
+          );
```

Comment:
Can you fix Fuchsia and add the web tests?

Reply:
Not sure what you mean by fix fuchsia?

Reply:
I simplified the options, ptal.

Reply:
Fuchsia icon should be set for `contains('_fuchsia')`. Previous filter was old based on legacy luci builders

Reply:
done